.gitlab-ci.yml

```diff
@@ -0,0 +1,26 @@
+stages:
+  - build
+
+variables:
+  CONTAINER_IMAGE: ergw/vpp-test
```

Comment:
why is this called vpp-test ?

Reply:
only for the test purpose , if there is no objection I will change the name to "vpp"